backend/resolvers/queries/scholarshipQuery.js

```diff
@@ -0,0 +1,36 @@
+import { UserError } from "graphql-errors";
+import { ADMIN, JUDGE } from "../../constants";
+import Scholarship from "../../models/scholarship";
+import PortfolioPeriod from "../../models/portfolioPeriod"
+
+export function scholarship(_, args, req) {
+
+  if (
+    req.auth.type !== ADMIN
+  ) {
+    throw new UserError("Permission Denied");
+  }
+
+  // Apply ordering, if desired
+  const order = args.orderBy
+    ? { order: [[args.orderBy.sort, args.orderBy.direction]] }
+    : {};
+
+  return Scholarship.findAll(order)
+}
+
+export function scholarshipByPeriod(_, args, req ) {
+
+  if (
+    req.auth.type !== ADMIN && req.auth.type !== STUDENT
+  ) {
+    throw new UserError("Permission Denied");
+  }
+
+  const periodId  = args.periodId;
+  
+  return PortfolioPeriod.findOne({
```

Comment:
This is the sequelize command that actually queries the database. The ```where``` property functions like a classic SQL where clause. The ```include``` property functions as a join, and will pull a list of entries from the ```Scholarship``` table that are associated with this ```PortfolioPeriod```. 

__However__ the linking table between ```PortfolioPeriods``` and ```Scholarships``` has not yet been created. Documentation on how to do this can be found [here](https://sequelize.org/master/manual/advanced-many-to-many.html)